### ISSUE TITLE
Different plasma protein binding for selected tissue

### ISSUE THREAD

**Issue by krinaj** [Open-Systems-Pharmacology/PK-Sim#592]
Hello,

I am wondering if there is a way to assign a different plasma protein binding parameter for the same molecule in a selected tissue (i.e. liver or gut) in PK-Sim?

thanks,
Krina

**Comment by PavelBal** [comment]
I do not think this is possible in PK-Sim. You have to go to MoBi.

**Comment by msevestre** [comment]
@krinaj I would recommend to ask questions in the forum. PK-Sim Issues are dedicated to bug fixing and feature request.

As @PavelBal mentioned, this is not possible in PK-Sim out of the box.
However depending on what your use case is, you could still implement it by for example adapting the Relative expression in the tissue of interest.

That being said, are you sure that you need to do this? What age are you trying to simulate?
Cheers,
Michael

**Comment by krinaj** [comment]
Thank you, Pavel and Michael. 

I was trying to simulate just normal adult human for ketoconazole. I saw in a publication where PBPK model from other software reported different fu for plasma protein and fu_gut, so was wondering if this can be replicated in PK-Sim. It sort of affected dose depending clearance.

Anyways, the dose dependent clearance can be better represented using MM kinetics and it worked well in PK-Sim when I tried estimating Kcat.

thanks,
Krina
